girder/web_client/src/collections/Collection.js

```diff
@@ -119,6 +132,21 @@ var Collection = Backbone.Collection.extend({
         return Math.ceil((this.offset - this.length) / this.pageLimit);
     },
 
+    /**
+     * Sets a specific pagenumber for loading by caluclating the offset
+     * @param {Number} pageNumber - the page that should be loaded based on the pageLimit size
```

Comment:
Ideally we'd follow jsdoc format for parameters.  This should be `@param {Number} pageNumber The page ...` (no hyphen before the description).